test/acceptance.go

```diff
@@ -86,7 +86,7 @@ func NewAcceptanceTest(t *testing.T, opts *AcceptanceOpts) *AcceptanceTest {
 func freeAddress() string {
 	// Let the OS allocate a free address, close it and hope
 	// it is still free when starting Alertmanager.
-	l, err := net.Listen("tcp", ":0")
+	l, err := net.Listen("tcp4", ":0")
```

Comment:
I guess with a normal port it defaults to ipv4 and with :0 to ipv6 then or something – doesn't matter.